src/main/java/com/xjeffrose/xio/client/loadbalancer/Distributor.java

```diff
@@ -32,11 +32,7 @@
   private final Timeout refreshTimeout;
 
   private final Ordering<Node> byWeight = Ordering.natural().onResultOf(
-      new Function<Node, Integer>() {
-        public Integer apply(Node node) {
-          return node.getWeight();
-        }
-      }
+    (Function<Node, Integer>) node -> node.getWeight()
```

Comment:
Did intellij generate this cast? You should be able to get away with 
```java
private final Ordering<Node> byWeight = Ordering.natural()
                                                .onResultOf((node) -> node.getWeight())
                                                .reverse();
```

Reply:
ya.  good ole intellij.  and i only touched that file to fix the javadoc errors...

Reply:
this didn't work.  reverted back to my original